packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/TableCellSelection.ts

```diff
@@ -84,6 +79,9 @@ export default class TableCellSelection implements EditorPlugin {
      * Dispose this plugin
      */
     dispose() {
+        if (this.firstTable) {
+            this.editor.select(this.firstTable, null);
```

Comment:
Do we always need to pass in the table object even we want to unselect?

Reply:
AH yes, we can just pass in a null parameter to remove the styles added if any.
Doing that change